Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,16 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const durationInHours = tasks.map(minute => minute.duration / 60);
+  const moreThan2Hours = durationInHours.filter(newArray => newArray >= 2);
+  const wageForTotalHours = moreThan2Hours.map(wage => wage * hourlyRate);
+  const totalWage = wageForTotalHours.reduce((total, num) => total + num);
+  return totalWage;
 }
 
-// eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-
-// add code to convert `earnings` to a string rounded to two decimals (euro cents)
-
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+const roundedEarnings = Number(earnings.toFixed(2));
```

Comment:
Excellent use of `.toFixed(2)` to round the number just prior to presenting the result to the user.